README.md

```diff
@@ -1,31 +1,39 @@
-Terraform Provider
-==================
+# Terraform Provider
 
-- Website: https://www.terraform.io
-- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+-   Website: <https://www.terraform.io>
+-   [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
+-   Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 
 <img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
-Requirements
-------------
+## Requirements
 
--	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
+-   [Terraform](https://www.terraform.io/downloads.html) 0.11.x
 
-Using the provider
-----------------------
-To use a released provider in your Terraform environment, run `terraform init` and Terraform will automatically install the provider.
+## Using the provider
+
+To use a released version provider in your Terraform environment, run `terraform init` and Terraform will automatically install the provider.  To update a provider to the latest version you can use `terraform init -upgrade` command.
```

Comment:
s/a released version/the latest version of the/ ?

Reply:
Good change.  I wasn't clear if the provider would auto-upgrade to the 2.x from 1.x without the `-upgrade` flag.